src/core/scene/a-scene.js

```diff
@@ -49,6 +50,7 @@ module.exports.AScene = registerElement('a-scene', {
     createdCallback: {
       value: function () {
         this.isIOS = isIOS;
+        this.isMacOS = isMacOS;
```

Comment:
do you want to create a `this.isApple`?

Reply:
Don't know what other Apple products like tvOS may do... if we have to go beyond just the two variants, then maybe.